templates/index.html

```diff
@@ -30,7 +30,7 @@
         {% else %}
     
             {% for note in notes %}
-                <form>
+                <form action="notes/note.id" method="delete">
```

Comment:
you need this to be the actual note id, not the string "note.id":  `action="/notes/{{note.id}}"`.

I don't think you're actually using this form, but since it's there it should have the right form.